modules/admin/admin-ui/src/main/resources/web/admin/apps/content-studio/js/app/wizard/ContentWizardPanel.ts

```diff
@@ -608,6 +608,17 @@ export class ContentWizardPanel extends api.app.wizard.WizardPanel<Content> {
 
             this.setSteps(steps);
 
+            // Content can be moved from its Site, but persisted content can still have metadata
+            // Such content needs update
+            const updateHandler = () => {
+                const needUpdate = !this.getPersistedItem().extraDataEquals(this.getExtraDataFromMetadata());
+                if (needUpdate) {
+                    this.saveChanges();
+                }
```

Comment:
I'm a bit unsure about this fix... First of all, we have to run this check every time we open any content (while the described case will happen very rarely). Second of all, I don't like that user gets a notification about saved content when he simply opens the wizard.
I understand that we need to clear metadata but can we do it in the backend when we move the content?